ChineseFilter/sections/specific.txt

```diff
@@ -26,6 +26,7 @@ mc.163.com###NIE-topBar-news
 soundofhope.org##.itb_ads
 soundofhope.org###itb_ads_catfish
 lihkg.com##iframe[src^="https://pb.lihkg.com/primis?"]
+niusnews.com##.fixed-footer-ad
```

Comment:
It leads to the https://www.niusnews.com/event/niustar/202204, so why did you add the rule to the AdGuard Chinese?
Annoyances - popups